save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/fileuploader/CosvFileManagerComponent.kt

```diff
@@ -0,0 +1,61 @@
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
+
+package com.saveourtool.save.frontend.components.basic.fileuploader
+
+import com.saveourtool.save.frontend.components.inputform.dragAndDropForm
+import com.saveourtool.save.frontend.utils.*
+import com.saveourtool.save.frontend.utils.noopLoadingHandler
+import com.saveourtool.save.utils.FILE_PART_NAME
+import js.core.asList
+import org.w3c.fetch.Headers
+import react.FC
+import react.Props
+import react.dom.html.ReactHTML.div
+import react.dom.html.ReactHTML.li
+import react.dom.html.ReactHTML.ul
+import react.useState
+import web.cssom.ClassName
+import web.file.File
+import web.http.FormData
+
+val cosvFileManagerComponent: FC<Props> = FC { props ->
+    useTooltip()
+
+    val (cosvAvailableFiles, setCosvAvailableFiles) = useState<List<String>>(emptyList())
+
+    val (filesForUploading, setFilesForUploading) = useState<List<File>>(emptyList())
+
+    val uploadCosvFiles = useDeferredRequest {
+        val uploadedFiles: List<String> = post(
```

Comment:
it's not uploaded files, it's uploaded ids

Reply:
Fixed